tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -594,14 +594,16 @@
     Type: custom,
     Tags: {
       attribute-bool: true,
-      attribute-boolArray: [false,true,false],
+      attribute-boolArray.0: false,
+      attribute-boolArray.1: true,
+      attribute-boolArray.2: false,
       attribute-boolArrayEmpty: [],
-      attribute-doubleArray: [4.1,5.0,6.0],
```

Comment:
This seems to have moved spans. Is that expected? 🤔

Reply:
FYI this moved to `metric` instead of `meta` in the same span as it's now just numbers instead of an array, so this is expected